deployment/ccip/changeset/solana_v0_1_0/cs_deploy_chain.go

```diff
@@ -236,9 +234,8 @@ func DeployAndMaybeSaveToAddressBook(
 	// the last bool is whether to overallocate the buffer account, if the program is going to be managed
 	// by timelock/mcms we want to overallocate the buffer account so that future upgrades can be performed
 	programID, err := chain.DeployProgram(e.Logger, cldf_solana.ProgramInfo{
-		Name:  programName,
-		Bytes: deployment.SolanaProgramBytes[programName],
```

Comment:
Qq, just because I lack context here... How are we deploying the binary if we're not passing the bytes along here?